### DISCUSSION TITLE
Authentication for Prometheus Metrics Provider

### DISCUSSION THREAD

**Discussion by ortwinschneider** [argoproj/argo-rollouts#1770] [Ideas]
I want to use AnalysisTemplate with Prometheus Metrics Provider and basic-auth. I think there is currently no way to configure basic-auth (or oAuth) and also use self signed certs.
When using the Prometheus rolled out with OpenShift ServiceMesh for example, it is automatically configured with basic-auth.

Are there any plans to support that?

**Comment by kostis-codefresh** [comment]
See https://github.com/argoproj/argo-rollouts/pull/5012 and https://github.com/argoproj/argo-rollouts/pull/4401
